src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/MercuryArticlePageObject.java

```diff
@@ -157,14 +186,193 @@ public void verifyCanonicalUrl() {
   public void verifyRepliesAreExpanded() {
     Assertion.assertTrue(checkIfElementOnPage(repliesContent.get(0)));
   }
-  
+
   public void verifySingleLinkedImageRedirect(int index) {
+    String methodName = "verifySingleLinkedImageRedirect";
     String currentUrl = driver.getCurrentUrl();
     singleImgLink.get(index).click();
     if (currentUrl.equals(driver.getCurrentUrl())) {
-      PageObjectLogging.log("verifySingleLinkedImageRedirect", "Redirection doesn't work", false);
+      PageObjectLogging.log(methodName, "Redirection doesn't work", false);
+    } else {
+      PageObjectLogging.log(methodName, "Redirection works", true);
+    }
+  }
+
+  public void verify25CommentsPerPage() {
+    String methodName = "verify25CommentsPerPage";
+    if (commentsList.size() == 25) {
+      PageObjectLogging.log(methodName, "There are 25 comments per comment page", true);
+    } else {
+      PageObjectLogging.log(methodName, "There aren't 25 comments per comment page", false);
+    }
+  }
+
+  public void verifyNextAndPreviousPageAreVisible() {
```

Comment:
Please refactor this method out of the nested if/else..
Consider using methods in the Assertion class and avoid nested if/else.